### DISCUSSION TITLE
Hubs Cloud AWS EU/Ireland: Mediaresolver (recticulum) crashes on ANY link on fresh stack installation, resulting in broken media links

### DISCUSSION THREAD

**Discussion by vvdt** [mozilla/hubs#5092] [Hubs Cloud Questions]
**Description**
Suddenly our hubs cloud instance can not resolve any web-link when placing an URL into the room, every link just resolves into a broken media link. These are just links to external websites, that normally work (including our own that only return text).

Update: our stacks are in EU/Ireland. Perhaps the problem is in the enterprise templates for this region?

To find the problem I tested this on two fresh new stack installation (new domain), both had the same issue right after a fresh installation. An older (personal edition) stack does still work, can't explain why. 

It seems the recticulum server returns a 500 error. When logging in the hubs application server I see the following error. This was by placing the link https://hubs.mozilla.com in the room. (The youtube errors are also present on the stack that still works, so not sure if they are causing the issue.)

![Screenshot 2022-01-03 at 15 32 34](https://user-images.githubusercontent.com/3598257/147943066-52d45440-3373-4689-b6f8-c074ee6d969d.png)

**To Reproduce**
Found in running stack with AWS template of Oct 2021. But you also reproduce with a fresh stack:
1. Install a new stack of hubs cloud on AWS
2. Choose Enterprise Multiserver or Single server (I did not test the personal edition yet)
3. Wait for the stack to be installed
4. Create a room (I did not
5. Use Place > Link to add a link. Use https://hubs.mozilla.com
6. You will get a broken media link, and if you access the application server using ssh, you will see the error from above.
 
**Expected behavior**
The link should be resolved (as it used to be), and not crash

**Screenshots**
![Screenshot 2022-01-03 at 15 39 19](https://user-images.githubusercontent.com/3598257/147943393-652df78d-36bc-4781-bbfa-0ce72c67180f.png)

**Hardware** 
 - Device: Macbook PRO, desktop
 - OS: MacOS
 - Browser: found with Brave, Firefox, Chrome

**Additional context**
Links and video's are a critical element in the rooms our clients make and use for teaching, so this issue is actually breaking for us..

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/HUBS-1236)

**Comment by vvdt** [comment]
Update: our stacks are in EU/Ireland. Perhaps the problem is in the enterprise templates for this region?

**Comment by yakyouk** [comment]
I confirm the exact same issue on a fresh deployment with the latest Enterprise Multiserver stack in region **Ireland**.

**Comment by vvdt** [comment]
Hi, is there perhaps somebody that can provide an update on this issue? Happy to help, but don't know where to start.

**Comment by yakyouk** [comment]
The issue is the update to nodejs 14 for Photomnemonic Serverless App. Everything else equal, Photomnemonic semantic version 0.2.0 (nodejs 10) works well, but it isn't possible anymore to deploy **new** Serverless Apps with nodejs 10.
